doc/rados/operations/erasure-code.rst

```diff
@@ -61,6 +61,31 @@ the following profile can be defined::
     $ rados --pool ecpool get NYAN -
     ABCDEFGHI
 
+Formula for calculating the raw size percentage overhead::
```

Comment:
Using the term "overhead" here is incorrect. As @jdurgin said, this ratio helps calculate the "effective space available", although when multiplying by 100, it gives the "percentage of effective space available".